examples/e2e-tests/kubernetes/node-count/50-nodes/definition.json

```diff
@@ -0,0 +1,39 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.8"
+    },
+    "masterProfile": {
+      "count": 5,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2",
+      "OSDiskSizeGB": 200
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 50,
```

Comment:
In the docs (https://github.com/Azure/acs-engine/blob/master/docs/kubernetes-large-clusters.md) ` We recommend the use of smaller pools (e.g., count of 20) over larger pools (e.g., count of 100); produce your desired total node count with lots of pools, as opposed to as few as possible.`  Would it make sense to change this to several smaller pools since people are likely to look at example files and to copy them for their own implementation?

Reply:
@amanohar @khenidak Is the above guidance still valid? (i.e., don't use node pools > 20 count)